test/e2e/e2e_test.go

```diff
@@ -16,7 +16,7 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"sigs.k8s.io/controller-runtime/pkg/client"
 
-	v1 "github.com/operator-framework/api/pkg/operators/v1"
+	operatorsv1 "github.com/operator-framework/api/pkg/operators/v1"
```

Comment:
nb: Just curious, what is the reasoning behind updating this name? To be more readable?

Reply:
Generally speaking, something like v1 or v1alpha1 is too ambiguous for me, so prepending the package name to the versioning can help improve readability + avoid duplicate package imports that also use a versioning directory structure. You'll commonly see things like `k8s.io/api/core/v1` and `k8s.io/apimachinery/pkg/apis/meta/v1` get aliased (respectively) as corev1 and metav1.